tempodb/tempodb.go

```diff
@@ -426,10 +426,15 @@ func (rw *readerWriter) EnablePolling(sharder blocklist.JobSharder) {
 		rw.cfg.BlocklistPollTenantIndexBuilders = DefaultTenantIndexBuilders
 	}
 
-	level.Info(rw.logger).Log("msg", "polling enabled", "interval", rw.cfg.BlocklistPoll, "concurrency", rw.cfg.BlocklistPollConcurrency)
+	if rw.cfg.TenantPollConcurrency <= 0 {
```

Comment:
i really don't like that these are all uints. you're just following suit so it's fine, but they should really be ints